x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/types.ts

```diff
@@ -83,7 +83,7 @@ export interface AlertTypeModel {
   id: string;
   name: string;
   iconClass: string;
-  validate: (alert: Alert) => ValidationResult;
+  validate: (alertParams: any) => ValidationResult;
```

Comment:
I think this `any` can be `IndexThresholdAlertParams`, no? 🤔

Reply:
yes :)